<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProxySettings</class>
 <widget class="QWidget" name="ProxySettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>207</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QFormLayout" name="mainLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="BuddyLabel" name="modeLabel">
     <property name="text">
      <string>Mode:</string>
     </property>
     <property name="buddy">
      <cstring>proxyMode</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="proxyMode"/>
   </item>
   <item row="1" column="0">
    <widget class="BuddyLabel" name="proxyTypeLabel">
     <property name="text">
      <string>Type:</string>
     </property>
     <property name="buddy">
      <cstring>proxyType</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="proxyType">
     <item>
      <property name="text">
       <string>HTTP Proxy</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>SOCKS Proxy</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="BuddyLabel" name="proxyHostLabel">
     <property name="text">
      <string>Host:</string>
     </property>
     <property name="buddy">
      <cstring>proxyHost</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="LineEdit" name="proxyHost"/>
   </item>
   <item row="3" column="0">
    <widget class="BuddyLabel" name="proxyPortLabel">
     <property name="text">
      <string>Port:</string>
     </property>
     <property name="buddy">
      <cstring>proxyPort</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="SpinBox" name="proxyPort">
     <property name="maximum">
      <number>65535</number>
     </property>
     <property name="value">
      <number>8080</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="BuddyLabel" name="proxyUsernameLabel">
     <property name="text">
      <string>Username:</string>
     </property>
     <property name="buddy">
      <cstring>proxyUsername</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="LineEdit" name="proxyUsername"/>
   </item>
   <item row="5" column="0">
    <widget class="BuddyLabel" name="proxyPasswordLabel">
     <property name="text">
      <string>Password:</string>
     </property>
     <property name="buddy">
      <cstring>proxyPassword</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="LineEdit" name="proxyPassword">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>lineedit.h</header>
  </customwidget>
  <customwidget>
   <class>SpinBox</class>
   <extends>QSpinBox</extends>
   <header>spinbox.h</header>
  </customwidget>
  <customwidget>
   <class>BuddyLabel</class>
   <extends>QLabel</extends>
   <header>buddylabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
